pkg/test/utils/kubernetes_test.go

```diff
@@ -537,7 +532,7 @@ func TestExtractGVKFromCRD(t *testing.T) {
 					Kind:    "testresource3",
 				},
 			},
-			shouldError: true,
+			shouldError: false,
```

Comment:
TBH I have no idea why this was true but anyway, it should not error in the current state